### DISCUSSION TITLE
Matterbridge: Add optional port number

### DISCUSSION THREAD

**Discussion by jedis00** [tteck/Proxmox#3786] [Ideas]
I made a [few updates](https://github.com/tteck/Proxmox/compare/main...jedis00:Proxmox:feature/matterbridge-port) and got as far as editing the installer scripts. But don't understand how to edit the main script that asks the questions during install. And also I'm not sure how to pass the user input port number into the script.

I confirmed the following is working by editing the service file on my local install, to use a custom port number.

## A clear and concise description of the feature request.

Add optional port number parameter to advanced setup:

https://github.com/tteck/Proxmox/blob/main/install/matterbridge-install.sh#L49

Add: ` -frontend NNNN` to end of line. Allow typing in custom port number

Example: ` -frontend 80` to run on port 80.

### What settings are you currently utilizing?

Advanced Settings

### Which Linux distribution are you employing?

Debian 12

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

```ExecStart=matterbridge -bridge -service```

### Please provide detailed steps to reproduce the issue.

Default port is 8283. Allow to use custom as listed above.

**Comment by MickLesk** [comment]
The question setting works with whiptail. In this case, however, it is absolutely unnecessary. I know that tteck also doesn't like it when there is too much data to choose from. It has to be easy to maintain and without causing headaches. 

If someone wants to adjust the port of the application, he can do that with a little brain. But I can hardly find a practical use for 90-95% of normal users.
